xml/XSLT/panelfile-2-9-6.xsl

```diff
@@ -150,15 +162,15 @@ This page was produced by <a href="http://jmri.org">JMRI</a>.
     <th>Out Sensors</th>
     <th>Comment</th>
     </tr>
-    <!-- index through individal route elements -->
+    <!-- index through individual route elements -->
     <xsl:apply-templates/>
     </table>
 </xsl:template>
 
 <xsl:template match="route">
 <tr>
-<td><xsl:value-of select="@systemName"/></td>
```

Comment:
The ``@`` selects the attribute. It was necessary to remove ``@`` as we moved away (deprecated in fact as stated in the types schema) the user name (and system name) from attribute to a separate element. This caused cells without content. Luckily the elements have the same names as the attributes, but that's the reason behind the change.